azkaban-execserver/src/main/java/azkaban/execapp/ServerStatisticsServlet.java

```diff
@@ -91,30 +91,70 @@ protected void fillRemainingMemoryPercent(ExecutorInfo stats){
       }
 
       long totalMemory = 0;
-      long  freeMemory = 0;
+      long totalFreeMemory = 0;
+      long parsedResult = 0;
 
       // process the output from bash call.
       // we expect the result from the bash call to be something like following -
       // MemTotal:       65894264 kB
-      // MemFree:        61104076 kB
-      if (output.size() == 2) {
+      // MemFree:        57753844 kB
+      // Buffers:          305552 kB
+      // Cached:          3802432 kB
+      // SwapCached:            0 kB
+      // Note : total free memory = freeMemory + cached + buffers + swapCached
+      if (output.size() == 5) {
```

Comment:
do we want an else statement for this if?
just in case we have a programmer error?